src/goSuggest.ts

```diff
@@ -184,6 +184,8 @@ export class GoCompletionItemProvider implements vscode.CompletionItemProvider {
 								}
 								item.insertText = suggest.name + '(' + paramSnippets.join(', ') + ') {{}}';
 							}
+							// Add same sortText to all suggestions from gocode so that they appear before the unimported packages
```

Comment:
Will they still be in the right order within the 'a' and 'z' groups, or should it be like `'a' + item.name`?